Modelica/Icons.mo

```diff
@@ -472,9 +472,9 @@ directly utilized by a user.
 This icon is designed for a <strong>rotational sensor</strong> model.
 </p>
 </html>"));
-  end RotationalSensor;
+  end RoundSensor;
 
-  partial class TranslationalSensor
+  partial class RectangularSensor
     "Icon representing a linear measurement device"
```

Comment:
The element's desription (and documentation) shall be modified accordingly. Whereby the `RotationalSensor` is "Icon representing a __round__ measurement device", the `RectangularSensor` is "... a __linear__ measurement device".

To be really honest, is is not clear at all when to use which icon for a sensor component.